src/sage/graphs/orientations.py

```diff
@@ -114,7 +114,7 @@ def acyclic_orientations(G):
     Graph with one edge::
 
         sage: list(Graph([(0, 1)]).acyclic_orientations())
-        [Graph on 2 vertices, Graph on 2 vertices]
+        [DiGraph on 2 vertices, DiGraph on 2 vertices]
```

Comment:
Actually, the output is `[Digraph on 2 vertices, Digraph on 2 vertices]`. Sorry for the confusion.